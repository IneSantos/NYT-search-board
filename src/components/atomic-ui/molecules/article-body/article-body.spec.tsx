import React from "react";
import { render } from "@testing-library/react";
import ArticleBody from "./article-body";

describe("ArticleBody component tests", () => {
  test(`ArticleBody component render`, () => {
    const { container } = render(
      <ArticleBody description={"This is an description"} webURL={"/"} />
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
