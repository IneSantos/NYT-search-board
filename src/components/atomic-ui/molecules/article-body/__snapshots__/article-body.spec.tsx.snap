// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleBody component tests ArticleBody component render 1`] = `
<div
  class="article-body"
>
  <div
    class="description"
  >
    This is an description
  </div>
  <a
    class="article-action article-action-Read the full article"
    href="/"
  >
     
    Read the full article
     
  </a>
</div>
`;
